src/components/main/Main.tsx

```diff
@@ -0,0 +1,127 @@
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../todoItem/TodoItem';
+import { TempTodo } from '../todoItem/TempTodo';
+import {
+  updateTodoComplited,
+  // updateTodoTitle,
+} from '../../api/todos';
+
+interface Props {
+  todos: Todo[] | null;
+  tempTodo: Todo | null;
+  showError: (errText: string | boolean) => void;
+  handleDeleteTodo: (id: number) => void;
+  loading: boolean;
+  loadingID: number;
+}
+export const Main: React.FC<Props> = ({
+  todos,
+  tempTodo,
+  showError,
+  handleDeleteTodo,
+  loading,
+  loadingID,
+}) => {
+  const handleUpdateTodoIsCompleted = async (
+    id: number,
+    complitedCurrVal: boolean,
+  ) => {
+    try {
+      await updateTodoComplited(id, {
+        completed: !complitedCurrVal,
+      // eslint-disable-next-line no-console
+      }).then(resp => console.log(resp));
```

Comment:
Remove the console.log statement from the code. Console logs should not be present in the production-ready code.